src/SmartComponents/ExperimentalCookie/ExperimentalCookie.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { instanceOf } from 'prop-types';
+import { Button } from '@patternfly/react-core';
+import { Main, Section, PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
+import { withCookies, Cookies } from 'react-cookie';
+
+class ExperimentalCookie extends Component {
+    static propTypes = {
+        cookies: instanceOf(Cookies).isRequired
+    };
+
+    constructor(props) {
+        super(props);
+        this.enableCookie = this.enableCookie.bind(this);
+        this.disableCookie = this.disableCookie.bind(this);
+    }
+
+    enableCookie() {
+        const { cookies } = this.props;
+
+        cookies.set('experimental-features', 'enabled', { path: '/' });
```

Comment:
can you break this out into constants.js? also I would call it `drift-experimental-features` so people know where the cookie came from :cookie: